scenes/Piece.gd

```diff
@@ -305,16 +303,41 @@ func _apply_player_input() -> void:
 			_reset_piece_target()
 			_calc_smush_target()
 			if _target_piece_pos != _piece.pos:
-				_move_piece_to_target()
-				$SmashSound.play()
-				_set_piece_state("_state_move_piece")
-				_piece.gravity = 0
-				_gravity_delay_frames = 4
+				_do_smush()
 	
 	if old_piece_pos != _piece.pos || old_piece_rotation != _piece.rotation:
 		if _piece.lock > 0 && !did_hard_drop:
 			_perform_lock_reset()
 
+"""
+Smushes a piece through other blocks towards the target.
+"""
+func _do_smush() -> void:
```

Comment:
Give this method a more suitable name in line with the method description, like "smush to target"

Reply:
Renamed method